audio/beets/Portfile

```diff
@@ -0,0 +1,517 @@
+
+    description     Plugin for the music library manager Beets. The mosaic plugin generates a montage of a mosiac from cover art.
+    long_description ${description}
+
+    homepage        https://github.com/susannamaria/beets-mosaic
+
+    checksums       rmd160  71d68c17894f255fcea01d6da07782e8eab9b324 \
+                    sha256  7e6ff14be69502c394dfc7abe57f144747adba700a2a50dd36c06e9a3e18a966 \
+                    size    6195
+
+    depends_lib-append \
+                    port:py${python.version}-fonttools \
+                    port:py${python.version}-parse \
+                    port:py${python.version}-pillow \
+                    port:py${python.version}-requests \
+                    port:py${python.version}-ttfquery
+}
+
+subport beets-noimport {
+    name            beets-noimport
+    version         0.1.0b5
+    revision        0
+
+    license         MIT
+
+    description     Beets plugin to add directories to the incremental import 'do not import' list
+    long_description ${description}
+
+    homepage        http://www.github.com/ttsda/beets-noimport
+
+    checksums       rmd160  2755bf7b969712da0931c5972ddf23f6aff7ec90 \
+                    sha256  86a6291e1de90b292829e6f7286b057be2bc556f5a49e9c30ffead2985ae998a \
+                    size    3167
+}
+
+subport beets-originquery {
+    PortGroup       github 1.0
+
+    github.setup    x1ppy beets-originquery 269e9e4cfc88128493b25ed814e34df2528dc1ef
+    version         20210908
+    revision        0
+
+    license         Unknown
+
+    description     Plugin for beets that improves album matching
+    long_description ${description}
+
+    homepage        https://github.com/x1ppy/beets-originquery
+
+    checksums       rmd160  6adbd6e65ef1a701af7daa7b137b6792ef4f8c12 \
+                    sha256  d015e1d21caba32ee02de184f71c12357c33e7b12d329976c1d1221bfa6f4033 \
+                    size    7563
+
+    depends_lib-append \
+                    port:py${python.version}-confuse \
+                    port:py${python.version}-jsonpath-rw \
+                    port:py${python.version}-yaml
+}
+
+subport beets-summarize {
+    PortGroup       github 1.0
+
+    github.setup    steven-murray beet-summarize 55ec5e1a839deedf62b815851cb8cc1e33cbfa61
+    name            beets-summarize
+    version         20210908
+    revision        0
+
+    license         LGPL-3
+
+    description     Summarize beet library statistics
+    long_description ${description}
+
+    homepage        https://github.com/steven-murray/beet-summarize
+
+    checksums       rmd160  bbafcc4edbb7262f2064b3099d0190afd93f12fc \
+                    sha256  22e3ba84c9777dc29110bde522dfc3ace1b69475607064d4c8e73a74e23ec74f \
+                    size    10966
+}
+
+subport beets-usertag {
+    PortGroup       github 1.0
+
+    github.setup    igordertigor beets-usertag 8e058361c9dfa335980048c0afb9274e5f077611
+    version         20210908
+    revision        0
+
+    license         Unknown
```

Comment:
according to upstream this should be `MIT`:
https://github.com/igordertigor/beets-usertag/blob/8e058361c9dfa335980048c0afb9274e5f077611/setup.py#L11`

can you please make sure

Reply:
@reneeotten you're right, I haven't checked `setup.py`.

Reply:
can you please change it then to `MIT` ;) ?